src/components/MovieDetails/index.tsx

```diff
@@ -85,7 +85,9 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
   const router = useRouter();
   const intl = useIntl();
   const { locale } = useLocale();
-  const [showManager, setShowManager] = useState(false);
+  const [showManager, setShowManager] = useState(
+    router.query.manage !== undefined ? true : false
+  );
```

Comment:
```suggestion
  const [showManager, setShowManager] = useState(
    !!router.query.manage
  );
```
Should work?

Reply:
But then we can't just use `?manage` to flag that the slideover should be opened. 😕  There isn't a really good way to handle boolean values, thus the check for `undefined`.